docs/fastdds/use_cases/use_cases.rst

```diff
@@ -1,15 +1,74 @@
 .. _typical_use_cases:
 
 Typical Use-Cases
-#################
+=================
 
-.. include:: includes/intro.rst
+.. _OSRF: https://www.openrobotics.org/
+.. _Robot Operation System 2 (ROS 2): https://index.ros.org/doc/ros2/
 
-.. include:: includes/fast-rtps_wifi.rst
+Fast DDS is highly configurable, which allows to use it in a large number of scenarios.
+This section provides configuration examples for the following typical use cases when dealing
+with distributed systems:
 
-.. include:: includes/wide_deployments.rst
++ :ref:`use-case-fast-rtps-over-wifi`.
+  Presents a case where :ref:`discovery` through multicast communication is a challenge.
+  This example shows how to:
 
-.. include:: includes/ros2.rst
+  - configure an initial list of peers with the address-port pairs of the remote participants
+    (see :ref:`use-case-initial-peers`).
+
+  - disable the multicast discovery mechanism (see :ref:`use-case-disabling-multicast-discovery`).
+
++ :ref:`wide_deployments`.
+  Describes a situation with a high number of deployed communicating agents.
+  In these kind of environments, Fast DDS recommends using one of the following configurations:
+
+  - the use of a centralized server for the :ref:`discovery` phases (see :ref:`server-client-discovery-use-case`).
```

Comment:
I know this used to be here, but actually I'd put it more on wifi cases

Reply:
You mean removing the `wide deploymens` altogether and fusing it with wifi?

Reply:
Sorry, I meant Just the Server-Client one